marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/test-data/entities/Customer.entity.json

```diff
@@ -0,0 +1,86 @@
+{
+  "info": {
+    "title": "Customer",
+    "version": "0.0.1",
+    "baseUri": "http://example.org/"
+  },
+  "definitions": {
+    "Customer": {
```

Comment:
Sorry to be picky here, but until we need all these properties, let's make these entity defs as minimal as possible. We always want to do that in tests, as it makes the focus of test that much more clear.

Reply:
I have renamed the entity models and included minimal properties.